src/shell.ts

```diff
@@ -155,6 +158,13 @@ export class Shell {
         await new Promise(r => setTimeout(r, 100))
         this.printPrompt()
     }
+
+    async escapeWatchdog() {
+        if (!this.watchdog) return
+        this.stopWatchdog()
+        if (terminal7.activeG)
```

Comment:
I prefer to use `terminal7` global :+1:

Reply:
What do you mean?

Reply:
There are places is the code where we use `this.t7` I don't like it. I prefer they way you did it here.